subgraphs/aave-v2/src/common/constants.ts

```diff
@@ -0,0 +1,106 @@
+import { BigDecimal, BigInt } from "@graphprotocol/graph-ts";
```

Comment:
Update this file with the latest `constants.ts` in the reference subgraph

Reply:
Updated 'common' directory to the current version. I had to comment out some of the functions in these files because of broken imports/references that do not apply to AAVE/Lending Protocols.

Reply:
Updated 'common' directory to the current version. I had to comment out some of the functions in these files because of broken imports/references that do not apply to AAVE/Lending Protocols.